cmake/modules/BuildDPDK.cmake

```diff
@@ -181,7 +181,8 @@ function(do_export_dpdk dpdk_dir)
   set_target_properties(dpdk::dpdk PROPERTIES
     INTERFACE_INCLUDE_DIRECTORIES ${DPDK_INCLUDE_DIR}
     INTERFACE_LINK_LIBRARIES
-    "-Wl,--whole-archive $<JOIN:${DPDK_ARCHIVES}, > -Wl,--no-whole-archive ${dpdk_numa} -Wl,-lpthread,-ldl")
+    "-Wl,--whole-archive $<JOIN:${DPDK_ARCHIVES}, > -Wl,--no-whole-archive ${dpdk_numa} -Wl,-lpthread,-ldl,-lmlx5")
+  target_link_libraries(dpdk::dpdk INTERFACE RDMA::RDMAcm IBVerbs::verbs)
```

Comment:
why do we *always* need to link against RDMA libraries when linking DPDK ?

Reply:
Because when the spdk and rdma compilation options are turned on, the RDMA library needs to be linked to pass the compilation.
After various attempts, I think it is more appropriate to link the rdma library here.